src/internals/ir/callGraph.ts

```diff
@@ -11,17 +11,32 @@ import {
   AstExpression,
   AstMethodCall,
   AstStaticCall,
+  AstContract,
+  AstId,
+  AstContractDeclaration,
 } from "@tact-lang/compiler/dist/grammar/ast";
 
-type CGNodeId = number & { readonly brand: unique symbol };
-type CGEdgeId = number & { readonly brand: unique symbol };
+export type CGNodeId = number & { readonly brand: unique symbol };
+export type CGEdgeId = number & { readonly brand: unique symbol };
+
+/**
+ * Flag constants for CGNode.
```

Comment:
I would comment what this specific flag means

Reply:
Add